src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/DoubleGenerator.java

```diff
@@ -0,0 +1,19 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+import java.util.Random;
+
+/**
+ * Created by lera on 27.12.16.
+ */
+public class DoubleGenerator extends NumberGeneratorImp<Double> {
+
+    @Override
+    public <Double extends Number> Double[] generateNumbers() {
```

Comment:
You shouldn't write generic type here it's defined in a class declaration. So `<Double extends Number>` is not needed.